.evergreen/config.yml

```diff
@@ -1666,6 +1713,16 @@ tasks:
             TOPOLOGY: "server"
         - func: "run enterprise auth tests"
 
+    - name: "test-search-index-helpers"
+      commands:
+        - func: "bootstrap mongo-orchestration"
+          vars:
+            VERSION: "rapid"
```

Comment:
I think this needs to be set to "latest" otherwise the unified tests are never run because rapid==6.3 and the tests require >=7.0.0.

Reply:
Done